libs/hdf-converters/src/asff-mapper/asff-mapper.ts

```diff
@@ -26,7 +29,10 @@ const COMPLIANCE_STATUS = 'Compliance.Status';
 
 // Sometimes certain ASFF file types require massaging in order to generate good HDF files.  These are the supported special cases and a catchall 'Default'.  'Default' files and non-special cased methods for otherwise special cased files do the pre-defined default behaviors when generating the HDF file.
 export enum SpecialCasing {
+  CMSInSpec = 'CMS Chef InSpec',
```

Comment:
Are these ASFF files that come from CMS running InSpec?

Reply:
Please see internal discussion